src/nvim/api/buffer.c

```diff
@@ -181,7 +181,8 @@ Boolean nvim_buf_attach(uint64_t channel_id,
   return false;
 }
 
-/// Deactivates buffer-update events on the channel.
+/// Deactivates buffer-update events on the channel. Not usable from lua,
+/// for attached lua callbacks see |api-buffer-updates-lua-detach|.
```

Comment:
not in VimL either. I guess this could be in the Attributes section of the docs.